data.json

```diff
@@ -426,10 +426,10 @@
     "name": "Gatsby",
     "difficulty": "hard",
     "description": "Get a pull request merged into Gatsby and receive a free shirt, socks, or stickers.",
-    "reference": "https://www.gatsbyjs.org/docs/contributor-swag/",
+    "reference": "https://www.gatsbyjs.com/contributing/contributor-swag/",
```

Comment:
It looks like the actual swag is Netlify swag, not Gatsby one. I think we want to update the Netlify entry instead and keep that one expired.

Reply:
sounds good, why don't we update as 'Gatsby x Netlify' swag?

Reply:
It's up to you

Reply:
fine then, I'II just update the name and date. instead of adding new swag.